test/testing/test_segmentation_scoring.py

```diff
@@ -62,15 +62,28 @@ def test_score_analysis_returns_expected_outputs_for_same_masks(self):
         assert precision == 1.0
         assert diffusion == 0.0
 
-    @pytest.mark.unit
+    @pytest.mark.antoine
     def test_score_analysis_runs_successfully_with_visualization_on(self):
-
+        tmp_folder = "/tmp"
```

Comment:
I think this folder might not work in Windows. Better off creating a temp directory relative to your current directory (e.g. "__tmp_segmentation_scoring__/" for it to be specific to this function), then deleting that entire directory at the end. At least then, if it doesn't get cleaned up, we'll know about it (instead of being dumped in a folder that's not version controlled). That's how I've been managing it anyways (I've had an issue where Windows would not let me delete files that I had explicitly created as "temp" files, using the [tempfile](https://docs.python.org/3/library/tempfile.html) module. If you feel strongly about using the `tmp/` folder, I could re-install a Windows virtual machine on my laptop and test it on there.

Reply:
Agreed. We should definitely use the tempfile module

Reply:
Sure, we can give it a go again, I'll just have to re-explore some workarounds for Windows, because some of its functionality was making tests failing on that OS back when I was first testing it out.

Reply:
let's do something simple because I'm going to refactor `score_analysis` so the metrics evaluation is separated from the visualization.

Reply:
How about not deleting temp file if permissions don’t allow? They will eventually be deleted by the OS at some point